Add Claws Mail, Planner, Nemo & PSequel (#264)
myfreeweb/awesome-gtk: README.md

### Email

- [Astroid](https://astroidmail.github.io) - Lightweight and fast Mail User Agent that provides a GUI to searching, displaying and composing email using [notmuch](https://notmuchmail.org) as backend `#c++` `#gtk3`.

> - [Claws Mail](https://claws-mail.org/) - Claws Mail is an email client (and news reader) `#c` `#gtk3`.

Reviewer: ```suggestion
- [Claws Mail](https://claws-mail.org) - Email client with plugin system to extend functionalities (news reader, calendar, etc.)  `#c` `#gtk3`.
```

### Database Clients

- [Daty](https://gitlab.gnome.org/World/Daty) - Cross-platform advanced Wikidata editor `#python` `#gtk3` `#libhandy`.

> - [PSequel](https://github.com/ppvan/psequel) - Modern, yet another Postgres SQL client `#vala` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [PSequel](https://github.com/ppvan/psequel) - Postgres SQL client `#vala` `#gtk4` `#libadwaita`.
```

- [Gnome Commander](https://gitlab.gnome.org/GNOME/gnome-commander) - Fast and powerful twin-panel file manager `#c++` `#rust` `#gtk3`.
- [Hyperplane](https://github.com/kra-mo/hyperplane) - Non-hierarchical file manager `#python` `#gtk4` `#libadwaita`.
- [GNOME Files (Nautilus)](https://apps.gnome.org/Nautilus) - Default file manager of the GNOME desktop `#c` `#gtk4` `#libadwaita` `#gnome`.

> - [Nemo](https://github.com/linuxmint/nemo/) - Default file manager of the Cinnamon desktop (fork of Nautilus) `#c` `#gtk3`.

Reviewer: ```suggestion
- [Nemo](https://github.com/linuxmint/nemo) - Default file manager of the Cinnamon desktop (fork of Nautilus) `#c` `#gtk3`.
```
